lib/console.js

```diff
@@ -64,9 +64,9 @@ Console.prototype.time = function(label) {
 
 
 Console.prototype.timeEnd = function(label) {
-  var time = this._times.get(label);
+  const time = this._times.get(label);
   if (!time) {
-    throw new Error('No such label: ' + label);
+    return;
```

Comment:
@ghaiklor ... would it be possible for you to add a `process.emitWarning(`No such label: ${label}`);` before returning here?